### PR TITLE
Added an option to send a copy of all generated e-mails to a bcc address.

### PR DESCRIPTION
**Pull request by hakakou**

### CODE REVIEW

File: app/Core/Mail/Transport/Mail.php [hunk 1]
```diff
@@ -36,6 +36,10 @@ public function sendEmail($recipientEmail, $recipientName, $subject, $html, $aut
                 ->setFrom($this->helper->mail->getMailSenderAddress(), $authorName)
                 ->setTo(array($recipientEmail => $recipientName));
 
+            if (defined('MAIL_BCC')) {
```
Comment (fguillot) [inline app/Core/Mail/Transport/Mail.php:39]: You should test if the value is not an empty string.

File: config.default.php [hunk 2]
```diff
@@ -44,6 +44,9 @@
 // E-mail address used for the "From" header (notifications)
 define('MAIL_FROM', 'replace-me@kanboard.local');
 
+// Bcc email for a copy of all notifications. Comment out if not needed.
+// define('MAIL_BCC', 'replace-me@kanboard.local');
```
Comment (fguillot) [inline config.default.php:48]: The default value should probably be an empty string and uncommented.

**Review by fguillot — COMMENTED** [review]
The new constant should also be defined here like others: https://github.com/kanboard/kanboard/blob/master/app/constants.php

**Comment by hakakou** [comment]
Thank you for your directions. I think I've implemented correctly.

**Comment by hakakou** [comment]
Sorry closed by mistake

**Pull request merged**